bin/kuboid-images.js

```diff
@@ -38,6 +38,10 @@ const table = new Table({
 
 for (const deployment of kubernetes.deployments()) {
     for (const container of deployment.containers) {
```

Comment:
can't you avoid using continue if you just pre-filter the collection first?

Aka: deployment.containers.filter(c => !c.startsWith('gcr.io/'))